pootle/apps/pootle_store/utils.py

```diff
@@ -316,16 +316,6 @@ def sub_unmute_qc(self, **kwargs):
         _kwargs.update(kwargs)
         return self.create_submission(**_kwargs)
 
-    def sub_create(self, **kwargs):
```

Comment:
the unitlifecycle class is not yet used but will be when i finish cleaning up subs